src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -604,6 +605,25 @@ private CallbackResponse prePopulateOrderDetailsPages(CallbackParams callbackPar
 
         updatedData.smallClaimsWitnessStatement(tempSmallClaimsWitnessStatement).build();
 
+        if (featureToggleService.isSdoR2Enabled()) {
+            SmallClaimsFlightDelay tempSmallClaimsFlightDelay = SmallClaimsFlightDelay.builder()
+                .smallClaimsFlightDelayToggle(checkList)
+                .relatedClaimsInput("In the event that the Claimant(s) or Defendant(s) are aware if other \n"
+                            + "claims relating to the same flight they must notify the court  \n"
+                            + "where the claim is being managed within 14 days of receipt of \n"
+                            + "this Order  providing all relevant details of those claims including \n"
```

Comment:
There are two spaces between "Order" and "providing"

Reply:
Two space removed